Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -140,8 +142,20 @@ YUI.add('ez-platformuiapp', function (Y) {
 
             this.on({
                 'loadingChange': this._loading,
-                'navigate': function () {
+                '*:contentDiscover': function() { this.set('isDiscovering', true)},
```

Comment:
hum, that's a bit too connected with the UDW, the app should not really know about those event, so the logic preventing the navigation should probably lie in the UDW view service and/or eZ.Plugin.UniversalDiscovery.